plugins/yaml/src/org/jetbrains/yaml/YAMLLanguageCodeStyleSettingsProvider.java

```diff
@@ -32,6 +27,41 @@
     };
   }
 
+  @NotNull
+  @Override
+  public CodeStyleConfigurable createConfigurable(@NotNull final CodeStyleSettings settings, @NotNull final CodeStyleSettings originalSettings) {
+    return new CodeStyleAbstractConfigurable(settings, originalSettings, YAMLLanguage.INSTANCE.getDisplayName()) {
```

Comment:
I think it would be better to reuse `YAMLCodeStyleSettingsProvider` here instead of inline it.

Reply:
Yeah, I was a bit unsure of the approach I would take. Will do as you suggest. Question though: does that mean that you alsow want the `codeStyleSettingsProvider` as part of `plugin.xml`? I looked through some of the other plugins and I didn't see a clear pattern on whether both classes are needed.

Reply:
Done. I made a guess that you wanted the minimal amount of changes necessary that would still solve the issue, so that is what I aimed for.